.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": ["prettier"],
+  "env": {
+    "es2023": true,
+    "browser": true
+  },
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2023,
+    "sourceType": "module"
+  },
+  "rules": {
+    "prettier/prettier": "error"
+  }
```

Comment:
since we're no longer using `eslint-plugin-prettier` just the config one; we can remove these lines

Reply:
I removed that